### PR TITLE
Bd read me

### PR DESCRIPTION
**Pull request by braddavistech**
Created readme file to show our project and how to download and run the files. Went through steps on a new computer to verify that the steps worked correctly. Created an mp4 then converted to gif to use on site for a preview.

### CODE REVIEW

**Comment by kellymorin** [comment]
👍 Looks good! We'll have a few more ownership comments to add once we complete the final merge,  but this is a great start. Thanks for getting this up to date.

File: src/scripts/landing.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
+// Author(s): Kelly Morin, Jase Hackman, Brendan McCray, Brad Davis
```
Comment (jasehackman) [inline src/scripts/landing.js]: I didn't help with this.

File: src/scripts/messages.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
+// Author(s): Kelly Morin, Brendan McCray, Brad Davis
```
Comment (jasehackman) [inline src/scripts/messages.js]: I helped with this one.

**Comment by braddavistech** [comment]
Verbal good to go from team.

**Pull request merged**